config/default.yml

```diff
@@ -198,6 +198,12 @@ PreCommit:
     required_executable: 'grep'
     flags: ['-IHn', '^<<<<<<<\s']
 
+  Pep257:
```

Comment:
I'm unsure about making this run by default (having never heard about this PEP until now).

Is this something people actually follow? What sort of adoption is there in the greater Python community?